Tools/WinMLDashboard/src/cpp/DebugRunner/DebugRunner.cpp

```diff
@@ -0,0 +1,63 @@
+#pragma once
+#include <windows.h>
+#include <winrt/Windows.AI.MachineLearning.h>
+#include <Windows.AI.MachineLearning.Native.h>
+#include "debugoperatorprovider.h"
+#include "debug_cpu.h"
+#include "BindingUtilities.h"
+
+using namespace winrt::Windows::AI::MachineLearning;
+
+class DebugRunner {
+public:
+
+	static bool Run(std::wstring modelPath, std::wstring inputPath, LearningModelDeviceKind kind)
+	{
+		std::wcout << modelPath;
+		auto customOperatorProvider = winrt::make<DebugOperatorProvider>();
+		auto provider = customOperatorProvider.as<ILearningModelOperatorProvider>();
+		auto model = LearningModel::LoadFromFilePath(modelPath, provider);
+		LearningModelSession session(model, LearningModelDevice(kind));
+		LearningModelBinding binding(session);
+
+		auto&& description = model.InputFeatures().GetAt(0);

```

Comment:
What if there are more than one inputs in the model.

Reply:
I think the input Path box in Run view need to handle multi-input case

Reply:
I think that handling multiple inputs would be good as well. I migrated most of the logic of the DebugRunner code over from WinMLRunner so perhaps we should have this discussion in regards to WinMLRunner functionality as well.